### ISSUE TITLE
Support field presence API in Proto 3

### ISSUE THREAD

**Issue by Dimezis** [salesforce/reactive-grpc#240]
When compiling protobufs with protoc-gen-rxgrpc and the new field presence API (optional modifier in Proto 3 messages), this error occurs:

```
Execution failed for task ':xxx:generateProto'.
> protoc: stdout: . stderr: yyy.proto: is a proto3 file that contains optional fields, but code generator protoc-gen-rxgrpc hasn't been updated to support optional fields in proto3. Please ask the owner of this code generator to support proto3 optional.--rxgrpc_out: 
```
 
https://github.com/protocolbuffers/protobuf/blob/master/docs/field_presence.md
https://github.com/protocolbuffers/protobuf/blob/master/docs/implementing_proto3_presence.md

Could you please update it?

**Comment by Dimezis** [comment]
I can drop a PR if you're accepting them at the moment

**Comment by Dimezis** [comment]
Ah, actually I see it's already implemented, but not released.
Is there any chance this could be released soon'ish?

**Comment by rmichela** [comment]
Duplicate of #230

**Comment by rmichela** [comment]
Released in 1.1.0
